drivers/SmartThings/zigbee-contact/src/battery-overrides/init.lua

```diff
@@ -0,0 +1,27 @@
+-- Copyright 2021 SmartThings
+--
+-- Licensed under the Apache License, Version 2.0 (the "License");
+-- you may not use this file except in compliance with the License.
+-- You may obtain a copy of the License at
+--
+--     http://www.apache.org/licenses/LICENSE-2.0
+--
+-- Unless required by applicable law or agreed to in writing, software
+-- distributed under the License is distributed on an "AS IS" BASIS,
+-- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+-- See the License for the specific language governing permissions and
+-- limitations under the License.
+
+local battery_defaults = require "st.zigbee.defaults.battery_defaults"
+
+local battery_range_2_1v_handler = {
+  NAME = "Cell battery 2.1v",
+  lifecycle_handlers = {
+    init = battery_defaults.build_linear_voltage_init(2.1, 3.0)
```

Comment:
Will this be taken care of automatically somehow on `added` as well?

Reply:
has to be in `init` (or any time before `added`) because that's when the zigbee driver base builds its cluster configurations.

Reply:
`init` is fired for every device know tho the driver at start up, is this the behavior that you are looking for?

It just strikes me as odd that you would want it to fire for a device init but not for a device add